test-integration/mock-backend-server/src/main/java/org.wso2am.micro.gw.mockbackend/MockBackEndServer.java

```diff
@@ -0,0 +1,451 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2am.micro.gw.mockbackend;
+
+
+import com.sun.net.httpserver.HttpServer;
+import io.netty.handler.codec.http.HttpHeaderNames;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.io.output.ByteArrayOutputStream;
+import org.json.JSONObject;
+
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.InetSocketAddress;
+
+import static org.wso2am.micro.gw.mockbackend.Constants.CONTENT_TYPE;
+import static org.wso2am.micro.gw.mockbackend.Constants.CONTENT_TYPE_APPLICATION_JSON;
+import static org.wso2am.micro.gw.mockbackend.Constants.MOCK_BACKEND_SERVER_PORT;
```

Comment:
Lets remove the static imports

Reply:
Fixed in ee03466e7304d4b68a1c39906fb6a12affb17948